backend/package.json

```diff
@@ -4,43 +4,39 @@
 	"description": "",
 	"main": "index.js",
 	"scripts": {
-		"build": "tsc",
-		"db:build": "serverless invoke local --function dbInit",
+		"db:init": "serverless invoke local --function dbInit",
```

Comment:
IMO there should be a drop and build function for dynamo and mysql as well as one that does both.

Reply:
Added redundancy